receiver/dockerstatsreceiver/metadata.yaml

```diff
@@ -0,0 +1,595 @@
+name: dockerstatsreceiver
+
+sem_conv_version: 1.6.1
+
+# Note: there are other, additional resource attributes that the user can configure through the yaml
+resource_attributes:
+  container.runtime:
+    description: "The runtime of the container. For this receiver, it will always be 'docker'."
+    type: string
+  container.id:
+    description: "The ID of the container."
+    type: string
+  container.image.name:
+    description: "The name of the docker image in use by the container."
+    type: string
+  container.name:
+    description: "The name of the container."
+    type: string
+  container.hostname:
+    description: "The hostname of the container."
+    type: string
+
+attributes:
+  core:
+    description: "The CPU core number when utilising per-CPU metrics."
+    type: string
+  device_major:
+    description: "Device major number."
+    type: string
+  device_minor:
+    description: "Device minor number."
+    type: string
+  interface:
+    description: "Network interface."
+    type: string
+
+metrics:
+  # CPU
+  container.cpu.usage.system:
+    enabled: true
+    description: "System CPU usage."
```

Comment:
Yes it does. Looking at the `moby` code, this is what it says: 

```
// getSystemCPUUsage returns the host system's cpu usage in
// nanoseconds. An error is returned if the format of the underlying
// file does not match.
```

source: https://github.com/moby/moby/blob/master/daemon/stats/collector.go#L123

This also raises another change we should make, because this metric is reported per-container, needlessly creating timeseries. In a later PR I can fix that.